src/lib/Microsoft.Health.Fhir.Ingest/Service/MeasurementEventNormalizationService.cs

```diff
@@ -88,6 +90,13 @@ private async Task StartConsumer(ISourceBlock<EventData> producer, IAsyncCollect
                             (DateTime.UtcNow - evt.SystemProperties.EnqueuedTimeUtc).TotalMilliseconds);
 
                         var token = _converter.Convert(evt);
+                        var tokenString = JsonConvert.SerializeObject(token);

```

Comment:
I've added a setting "NormalizationService:LogDeviceIngressSizeBytes" to make this optional. It will default to false. Let me know if you have feedback on how this setting is injected. I tried to follow the approach extension/bindings/attribute as opposed to just passing a boolean in.